source/faq.txt

```diff
@@ -107,3 +107,20 @@ have more than 2 non-arbiter nodes and you must sync from a non-arbiter
 node. Use the source cluster's connection string to specify a
 :ref:`read preference <mongodb-uri>` for a non-arbiter, data-bearing
 node. 
+
+What If I see a Slow Operation Warning?
```

Comment:
[suggestion]

Not totally sure, but I think "if" should be lower case because it's a short conjunction, and "See" should be capitalized because it's a verb?

These are guidelines I found in the APA style guide: https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

Reply:
It looks wrong to me, but I'm probably reading it as Latin in my head.  Changed to "if".